examples/step-30/step-30.cc

```diff
@@ -918,8 +885,8 @@ namespace Step30
             }
           // Now we analyze the size of the mean jumps, which we get dividing
           // the jumps by the measure of the respective faces.
-          double average_jumps[dim];
-          double sum_of_average_jumps = 0.;
+          std::array<double, dim> average_jumps;
```

Comment:
not wrong, but why?

Reply:
Its always good to advertise modern features!